### ISSUE TITLE
How to display a calculated value as stars

### ISSUE THREAD

**Issue by phuclh** [nikaia/nova-rating-field#2]
I need to display a star rating with an average rating, then I try to pass that average value into `resolvingUsing` or `displayUsing` function but it didn't work.

```
Rating::make('Rating')->resolveUsing(function ($name) {
                return $this->articles->avg('rating');
            })
```

How can I display a calculated value as stars (that data isn't a column in the table)

**Comment by phuclh** [comment]
@nbourguig do you have any ideas for that? Thank you.

**Comment by nbourguig** [comment]
Maybe related to #1, fixed in v1.0.2
